src/Context.cpp

```diff
@@ -0,0 +1,21 @@
+#include "Context.h"
+
+#include "licenses/LicenseCategory.h"
+
+Context::Context() {
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelWebView, licenses::LicenseCategory::WebView);
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelToolkit, licenses::LicenseCategory::Toolkit);
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelPlatform, licenses::LicenseCategory::Platform);
+  connectSearchModel(&m_licenseModel, m_filterLicenseModelAll, licenses::LicenseCategory::All);
+}
+
+void Context::connectSearchModel(licenses::LicenseModel * model, licenses::LicenseFilter & search_model, licenses::LicenseCategory category) {
```

Comment:
a reference instead of a pointer might be better, or is that required in QML?

Reply:
I changed the parameter to a reference